target_finder/classification.py

```diff
@@ -109,6 +109,8 @@ def _do_classify(blob, min_confidence):
 
     cropped_img = blob.image
 
+    primary, secondary, alpha_img = _get_color(blob)
```

Comment:
I think it'd be good to bring more of the logic up to the `_do_classify` function, since returning the alpha as a side affect of getting the colors is a little odd. So like you could have a function that just returns the two main hex codes (I think that's what they are?), do the alpha extraction which is presumably for future classification, and then just give the label the color names afterwards.

Reply:
And I guess at that point the `_get_color` function is really just acting as a wrapper for the `_get_main_colors` function with the count check.